dotCMS/src/main/java/com/dotmarketing/quartz/DotStatefulJob.java

```diff
@@ -1,7 +1,31 @@
 package com.dotmarketing.quartz;
 
+import java.util.Date;
+import org.quartz.JobDetail;
+import org.quartz.SimpleTrigger;
 import org.quartz.StatefulJob;
+import org.quartz.Trigger;
 
 public abstract class DotStatefulJob extends DotJob implements StatefulJob {
 
+    public static final class TriggerBuilder {
+        private JobDetail jobDetail;
```

Comment:
Codacy found an issue: [Field jobDetail has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5971105)